packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec

```diff
@@ -5,8 +5,8 @@
 %global gem_name activerecord-nulldb-adapter
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.0
-Release: 2%{?dist}
+Version: 0.7.0
```

Comment:
why not 0.8.0?

Reply:
On github the latest version is 0.7.0, adding 0.8.0 now